1-exercises/A-array-find/exercise.js

```diff
@@ -5,6 +5,11 @@
 
 // write your code here
 
+function findName(names){
+  let longNameThatStartsWithA = names.find(name => name.startsWith('A') && name.length > 7);
+  console.log(longNameThatStartsWithA);
```

Comment:
You are logging the function. We need to call the function

Reply:
I call the `findName` function and pass in the `names` array as an argument.